addons/web/static/src/core/alert/alert.scss

```diff
@@ -0,0 +1,4 @@
+.alert {
```

Comment:
Shouldn't we specify another class name for this king of alert? 
I don't realize how many inputs are in alerts.

Reply:
Anyway, we should move this to bootstrap_review.scss